widgets/form/src/App.vue

```diff
@@ -151,6 +165,86 @@ fetch(props.url)
     .catch((error) => {
         console.error(`ASSIST Embed Form ${error}`);
     });
+
+async function authenticate (formData, node) {
+    node.clearErrors();
```

Comment:
Just curious, what is this for?

Reply:
As far as I know, it will clear all previous errors before adding the new ones from the current submission. I think this was already part of the form submission handling before I started? Or maybe Copilot suggested it.